pkg/image/utilities/utilities.go

```diff
@@ -595,7 +619,11 @@ for ($i = $args.Length; $i -ge 0; $i--)
             $vals = $null
         }
         '^(-r|--registry)$' {
-            $registry = ($vals | Select-Object -First 1)
+            $target_registry = ($vals | Select-Object -First 1)
+            $vals = $null
+        }
+        '^(-r|--source-registry)$' {
```

Comment:
doesn't this imply `-r` is a shorthand for both `target_registry` and `source_registry`? `-r` will never match `--source-registry` here since it will exit above.